services/input-service.js

```diff
@@ -150,11 +154,12 @@ class InputService {
     }
 
     getScoreFunctionId(options) {
-        return (
-            options.scoreFunctionId ??
-            this.config.scoreFunctionId ??
-            DEFAULT_PARAMETERS.SCORE_FUNCTION_ID
-        );
+        const environment =
+            options.environment ?? this.config.environment ?? DEFAULT_PARAMETERS.ENVIRONMENT;
+        console.log(environment);
```

Comment:
Can you please remove console.log

Reply:
good catch, sorry for that, will do right away